cmd/dp-file-downloader/main.go

```diff
@@ -8,7 +8,7 @@ import (
 	"os/signal"
 	"syscall"
 
-	healthcheck "github.com/ONSdigital/dp-api-clients-go/health"
+	healthcheckapi "github.com/ONSdigital/dp-api-clients-go/health"
```

Comment:
I'm not sure about this name change - it's not an api, but a healthcheck for other apis. What complaint did the linter have about it being called healthcheck?

Reply:
The linter complained that healthcheck was defined here on line 11 as an import and then also on line 61 as a var declaration. Had to change one of the names.

Reply:
Maybe `healthcheckclient` then